homeassistant/components/mycroft/notify.py

```diff
@@ -24,16 +25,17 @@ def get_service(
 class MycroftNotificationService(BaseNotificationService):
     """The Mycroft Notification Service."""
 
-    def __init__(self, mycroft_ip):
+    def __init__(self, mycroft_ip) -> None:
         """Initialize the service."""
         self.mycroft_ip = mycroft_ip
+        self.bus = MessageBusClient(host=self.mycroft_ip)
```

Comment:
Please create the bus and start the thread in `get_service`. Then just pass the bus instance to `MycroftNotificationService`.